tests/test_router_bgp.rb

```diff
@@ -324,10 +324,6 @@ def bestpath_not_configured(bgp)
     if platform == :nexus
       refute(bgp.bestpath_med_non_deterministic,
              'bgp bestpath_med_non_deterministic should *NOT* be enabled')
-    else
-      assert_raises(Cisco::UnsupportedError) do
-        bgp.bestpath_med_non_deterministic
```

Comment:
on XR you may want to assert that it returns nil, rather than not testing it at all?